### ISSUE TITLE
Implement plugin - colorpicker

### ISSUE THREAD

**Issue by johnalarcon** [ClassicPress-research/ClassicPress-Editor#32]
### Implement `colorpicker` TinyMCE plugin

The various controls and behaviors of the v4 plugin currently included in ClassicPress should be reasonably replicated by the v5 editor, or by the updated TinyMCE core. Prior to testing, install the [ClassicPress Editor](https://github.com/ClassicPress-research/ClassicPress-Editor/archive/refs/heads/main.zip) plugin. **Do not activate it.** If the plugin is already installed, be sure to deactivate it before starting your tests.

#### Testing Notes
```
1. Click to edit any post with the built-in editor.
2. Add the element being tested to the post body using the relevant editor button/menu.
3. Save the post.
4. Activate the new Classic Editor plugin.
5. Click to edit the same post.
6. Verify that the new editor replicates the expected behavior.
```

This issue has a corresponding item on the Task List in issue #2.

**Comment by joyously** [comment]
When I use the default core configuration (as in my PR), I get
`Color picker plugin is now built in to the core editor, please remove it from your editor configuration`
That's why I removed it.
I didn't understand why the v5 download comes with that plugin, if it's built in.

**Comment by johnalarcon** [comment]
As on another issue: Yes, that's correct: it's now built in and I've updated the Tiny config to reflect this. The code has not yet been uploaded.

**Comment by joyously** [comment]
The `colorpicker` folder has been deleted in 02c5fbc.

TinyMCE tries to load whatever is in the plugins config parameter, so this plugin was removed from the list in 947be9.
